README.md

```diff
@@ -1,11 +1,11 @@
-# SimpleKeychain
+# SimpleKeychain (First Availability)
```

Comment:
I would consider linking the reader to the product lifecycle docs so that they can better understand what "First Availability" means.

https://auth0.com/docs/troubleshoot/product-lifecycle/product-release-stages